os/drivers/fs_manager/Kconfig

```diff
@@ -0,0 +1,10 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config FS_MANAGER_DRV
```

Comment:
You add this to enable a driver for file system operation. But you don't make a dependancy between fscmd, fs tc and this. You should do it as well.